src/ripple/app/main/Main.cpp

```diff
@@ -288,7 +288,6 @@ int run (int argc, char** argv)
     ("conf", po::value<std::string> (), "Specify the configuration file.")
     ("rpc", "Perform rpc command (default).")
     ("rpc_ip", po::value <std::string> (), "Specify the IP address for RPC command. Format: <ip-address>[':'<port-number>]")
-    ("rpc_port", po::value <std::uint16_t> (), "Specify the port number for RPC command.")
```

Comment:
I'm not super comfortable with removing the `--rpc_ip` option without any warning. I have a hunch that this is used more widely than we might assume in production systems. I'd suggest we deprecate it, have it output a warning (to `cerr`?) if used, but keep it in until 1.0. (Of course, I could be wrong.)

You could maintain backward compatibility by populating a local var from the `--rpc_port` option, then using that value if it's set and `res.first.port() == 0`.

Reply:
good point - reinstated the flag, but will only look for it is none is provided in `rpc_ip`.